tests/Logging_.py

```diff
@@ -61,31 +61,14 @@ def testLoggingWithBasicAuthentication(self):
         self.assertEqual(
             github.Github(self.login, self.password).get_user().name, "Vincent Jacques"
         )
-        # In Python 3.4+, dicts are not output in the same order as in Python 2.7.
```

Comment:
Bad! This is unrelated. :-) However, I noticed this monstrosity thanks to this, so I've pushed up https://github.com/PyGithub/PyGithub/pull/1315. I've run black over my changes, so it shouldn't impact your black CI changes.

Reply:
@sfdye Shall I merge in that refactoring and you can rebase this to not lose any test coverage?

Reply:
👍